modules/geolocation/geolocation.py

```diff
@@ -0,0 +1,99 @@
+"""
+Geolocation module to map pixel coordinates to geographical coordinates
+"""
+
+import numpy as np
+
+class Geolocation:
+    """
+    Locates the geographical position of a set of pixels
+    """
+
+    # TODO Class members
+    def __init__(self):
+        """
+        Initial setup of class members
+
+        Returns
+        -------
+        Geolocation
+        """
+
+        # Input to gather_point_pairs()
+        self.__cameraOrigin3o = np.array([0.0, 0.0, 2.0])
```

Comment:
What's with these values for cameraorigin and cameradirection? As in why aren't they 0? Is it just a value so we can see where it should face?

Reply:
They're currently random defaults, I can set them to zero if you want.

Reply:
Gotcha, leave it as it is then